invadedhouse/models/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .chest import Chest
+from .house import House
+from .inventory import Inventory
+from .item import Item
+from .monster import Monster
+from .player import Player
+from .room import Room
+from .weapon import Weapon
+
+__all__ = [
+    'Chest', 'House', 'Inventory', 'Item',
+    'Monster', 'Player', 'Room', 'Weapon'
+]
```

Comment:
folders in python are also called `packages`. There is a special file called `__init__.py` which tells Python how to initialize the package. Again, this is more advanced stuff that will come in handy later - for now just trust me that this file makes the program run smoother.

Reply:
whats going on up there.... WHOAH  imports all over

Reply:
this is a shortcut to make it easier to import classes from other files. So instead of:
```
from models.chest import Chest
```

you can just do
```
from models import Chest
```

Not a big deal right now but when the project gets a lot larger it's just nicer. :)